ci/perf_trace_decode.py

```
@@ -0,0 +1,157 @@
+FRAME_MAGIC = 0xF8AAE000
+
+EVENT_FMT = "<IhhIII"  # magic, depth (i16), line (i16), dur_us, start_us, name_offset
+EVENT_SIZE = struct.calcsize(EVENT_FMT)
+EVENT_MAGIC = 0xE0E07E07
+
+
+def decode(blob: bytes) -> dict[str, Any]:
+    if len(blob) < HEADER_SIZE:
+        raise ValueError(f"too short ({len(blob)} bytes) for header")
+
+    magic, version, num_frames, total_bytes, _ = struct.unpack(
```

Ruff (0.15.13)

[warning] Unpacked variable `total_bytes` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)